src/main/java/com/github/manolo8/darkbot/core/manager/PetManager.java

```diff
@@ -28,18 +28,12 @@
 import eu.darkbot.api.managers.PetAPI;
 import eu.darkbot.api.utils.Inject;
 import eu.darkbot.api.utils.ItemNotEquippedException;
+import eu.darkbot.util.TimeUtils;
+import eu.darkbot.util.Timer;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.EnumMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
avoid wildcard imports, edit your IDE settings to not collapse imports into wildcards and revert these